plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry exported="true" kind="lib" path="lib/commons-compress-1.8.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/jboss-dmr-1.2.0.Final.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/log4j-1.2.17.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/openshift-java-client-3.0.0-SNAPSHOT.jar" sourcepath="/openshift-java-client"/>
-	<classpathentry exported="true" kind="lib" path="lib/slf4j-api-1.6.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/slf4j-log4j12-1.6.1.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
```

Comment:
@adietish I purposely removed this file.  ".classpath" and ".project" files will be regenerated when importing a project into eclipse and they are typically user specific.  I find that with the fiddling I do with these plugins that these files sometime change and are identified as needing to be updated in Git event though they are files we really shouldn't be checking in anyway.